internal/pkg/webconfig/webconfig.go

```diff
@@ -217,7 +217,7 @@ func (s *Server) Run(ctx context.Context) ([]byte, error) {
 	}
 
 	go func() {
-		if err = s.svr.ListenAndServe(); err != nil {
+		if err = http.ListenAndServe(":80", http.HandlerFunc(redirect)); err != nil {
```

Comment:
Kept it simple and just setup a listener for port 80 that knows how to hand redirects.